lib/mayaUsd/ufe/UsdUndoDuplicateSelectionCommand.cpp

```diff
@@ -70,24 +60,46 @@ UsdUndoDuplicateSelectionCommand::Ptr UsdUndoDuplicateSelectionCommand::create(
     const Ufe::Selection&       selection,
     const Ufe::ValueDictionary& duplicateOptions)
 {
-    auto retVal = std::make_shared<UsdUndoDuplicateSelectionCommand>(selection, duplicateOptions);
-    if (retVal->_perItemCommands.empty()) {
-        // Could not find any item from this runtime in the selection.
-        return {};
+    bool canDuplicate = false;
+    for (auto&& item : selection) {
+        UsdSceneItem::Ptr usdItem = std::dynamic_pointer_cast<UsdSceneItem>(item);
+        if (usdItem) {
+            canDuplicate = true;
+            break;
```

Comment:
[Comment] Would it be interesting to continue the loop to remove null `usdItem` from the list?

Reply:
.. allowing to remove lines 88 to 90 & perhaps right away create a list/set of `UsdSceneItem::Ptr`

Reply:
Maybe. Anyway this will be revisited at some point to fix MAYA-125854 and we will also remove all usdItems that have a parent in the list.

Reply:
OK, that was a great idea and it also provided a very efficient way to fix MAYA-125854. Done.